util.mli

```diff
@@ -72,9 +72,15 @@ val ( |> ) : 'a -> ('a -> 'b) -> 'b
   (** Forward pipe operator; useful for composing JSON access functions
       without too many parentheses *)
```

Comment:
I haven't tested this but ocamldoc supports `@deprecated`. I guess you'd place it at the beginning of the comment or something. See http://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html